test/blackbox-tests/test-cases/melange/private.t/run.t

```diff
@@ -0,0 +1,24 @@
+Test the private libs flow when using `melange.emit` stanza
+
+Cmj rules should include --bs-package-output
+  $ dune rules inside/app/.app.objs/melange/app.cmj | 
+  > grep -e "--bs-package-output" --after-context=1 
+      --bs-package-output
+      _build/default/inside/app
+
+Cmj rules should not include --bs-package-name
+  $ dune rules inside/app/.app.objs/melange/app.cmj | 
+  > grep -ce "--bs-package-name"
+  0
+  [1]
+
+Js rules should include module type
+  $ dune rules inside/output/app/app__B.js | 
+  > grep -e "--bs-module-type" --after-context=1 
+      --bs-module-type
```

Comment:
Is there a way to get all this data from inspecting the .js produced perhaps? it's a bit fragile to use `dune rules`.

Reply:
hm, the module type yes. But the ` --bs-package-output` and `--bs-package-name` are harder to check.

For cases like this, I miss some intermediate `display` mode in dune that is just like `short` but including the full command.